pkg/detectors/jdbc/sqlserver_integration_test.go

```diff
@@ -35,7 +35,7 @@ func TestSqlServer(t *testing.T) {
 			wantPing: true,
 		},
 		{
-			input:    "//localhost;database= master;spring.datasource.password=" + sqlServerPass,
```

Comment:
The SQL Server detector does not appear to accept any connection strings with spaces. (This should probably be fixed!)